homeassistant/components/onewire/__init__.py

```diff
@@ -4,15 +4,22 @@
 
 from pyownet import protocol
 
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
 from homeassistant.helpers import device_registry as dr
 
-from .const import DOMAIN, PLATFORMS
+from .const import DOMAIN
 from .onewirehub import CannotConnect, OneWireConfigEntry, OneWireHub
 
 _LOGGER = logging.getLogger(__name__)
 
+_PLATFORMS = [
```

Comment:
Why the `_`

Reply:
Because it's private, and shouldn't be imported/used elsewhere